Sources/Tentacle/User.swift

```diff
@@ -77,7 +77,7 @@ public struct UserInfo {
     public let email: String?
     
     /// The URL of the user's website if they've set one
-    public let websiteURL: URL?
+    public let websiteURL: String?
```

Comment:
Want to add a note to the documentation that clarifies that this can be any string?